.github/workflows/cli-k3s-multi_cluster-rm_stable.yaml

```diff
@@ -0,0 +1,41 @@
+# This workflow calls the master E2E workflow with custom variables
+name: CLI-K3s-Multi_Cluster-RM_Stable
+
+on:
+  # !! DEBUG !!
+  pull_request:
+  # !! DEBUG !!
+  workflow_dispatch:
+    inputs:
+      destroy_runner:
+        description: Destroy the auto-generated self-hosted runner
+        default: true
+        type: boolean
+  schedule:
+    # Every Sunday at 3am
+    - cron: '0 3 * * 0'
+
+jobs:
+  cli:
+    uses: ./.github/workflows/master-e2e.yaml
+    secrets:
+      credentials: ${{ secrets.GCP_CREDENTIALS }}
+      pat_token: ${{ secrets.SELF_HOSTED_RUNNER_PAT_TOKEN }}
+      slack_webhook_url: ${{ secrets.SLACK_WEBHOOK_URL }}
+    with:
+      test_description: "CI/Manual - CLI - MultiCluster - Deployment test with Standard K3s"
+      cluster_name: cluster-k3s
+      cluster_number: 40
+      # !! DEBUG !!
+      #destroy_runner: ${{ github.event_name == 'schedule' && true || inputs.destroy_runner }}
+      # !! DEBUG !!
+      destroy_runner: false
+      iso_boot: true
+      k8s_version_to_provision: v1.26.8+k3s1
+      # !! DEBUG !!
+      operator_repo: oci://registry.suse.com/rancher
+      os_to_test: stable
+      # !! DEBUG !!
+      runner_template: elemental-e2e-ci-runner-spot-x86-64-template-n2-highmem-64-v4
+      test_type: multi
```

Comment:
As we use this variable to difference `ui` and `cli` tests, could you use `multi_cli` instead?

Reply:
Yes, I split to `single_cli` and `multi_cli` (in addition to the already existing value `ui`).